qemu/tests/cfg/vdpa_sim_blk_test.cfg

```diff
@@ -0,0 +1,45 @@
+- vdpa_sim_blk_test:
+    type = vdpa_sim_blk_test
+    virt_test_type = qemu
+    start_vm = no
+    only x86_64 i386
+    vdpa_sim_blk_images = "stg1 stg2"
+    images += " ${vdpa_sim_blk_images}"
+    image_format_stg1 = raw
+    image_raw_device_stg1 = yes
+    image_format_stg2 = raw
+    image_raw_device_stg2 = yes
+    drive_format_stg1 = virtio
+    drive_format_stg2 = scsi-hd
```

Comment:
Do we need to cover virtio_blk as system image? If yes, we need to add a scsi-pci when hotplugging stg2. e.g. drive_bus_stg2 = 4, Otherwise, the case will fail to run.  error message: {'class': 'GenericError', 'desc': "No 'SCSI' bus found for device 'scsi-hd'"}

Reply:
Yes, updated. thanks.